src/plugins/data_table/data_table.js

```diff
@@ -100,6 +100,11 @@ var LocalDataTable = (function() {
       return this.dataTable.fnSettings().fnRecordsTotal();
     },
 
+    getPageLimit: function() {
+      this._lockManager.ensureUnlocked("bulk");
+      return this.dataTable.fnSettings()._iDisplayLength;
+    },
```

Comment:
this looks like the only accessor using `get...` in the name.  The others are just the noun, like `totalRecordsCount` and `columnVisibility` etc.

What about `pageLimit`?